docs/docs.less

```diff
@@ -14,8 +14,16 @@
 */
 @import (reference) 'vars';
 
-body {
-  background-color: @gray-25;
+.brand-logo {
```

Comment:
Pulled in from `src/helix-ui/styles/layouts.less` (older implementation of layouts)